web_external/InfoView/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react';
+
+import './style.styl';
+
+class InfoView extends PureComponent {
+    constructor(props) {
+        super(props);
+    }
+    render() {
+        return <div className={['v-infoview', this.props.className].join(' ')}>
+            <div className='panel panel-default'>
+                <div className='panel-heading'>Info</div>
+                <div className='panel-body'>
+                    {this.props.annotations && this.props.annotations.length &&
+                        <div>
+                            <ul className='geometry'>
+                                {this.props.annotations.map((annotation) => {
+                                    return <li key={annotation.geometry.id0} className='track'>
+                                        <div>
+                                            <div>{annotation.track.obj_type} {annotation.track.id1}</div>
+                                            <div>Geometry id: {annotation.geometry.id0}</div>
+                                            <div>Frame id: {annotation.geometry.ts0}</div>
+                                            {Object.entries(annotation.geometry.keyValues).map(([key, value], index) => {
+                                                return <div key={index}>{key}: {value}</div>
+                                            })}
+                                            {annotation.activities && annotation.activities.length !== 0
+                                                && <ul className='activity'>
+                                                    {annotation.activities.map((activity) => {
+                                                        return <li key={activity.id2} className='activity'>
+                                                            <div>Activity Id: {activity.id2}</div>
+                                                            <div>Activity: {activity.act2}</div>
+                                                        </li>
+                                                    })}
+                                                </ul>}
+                                        </div>
+                                    </li>
+                                })}
+                            </ul>
+                            <div className='clear-message'>(click outside annotation to clear)</div>
```

Comment:
Suggestion: "Click an annotation to view details. Click on an un-annotated  space to clear."

Reply:
I think it's too long to to fit nicely. However, any single line alternative is welcomed.

Reply:
Why does it need to be a single line?

Reply:
> Firefox does. I tested with it, and it causes Bad Things to happen. Moreover, I have a philosophical objection to preventing the user from changing their text size and/or unnecessarily penalizing users that do so.
> 
> Using px exclusively instead of text-based units where appropriate is basically saying "(redacted) you" to users that might want or need to use larger text.

lets use px for now and discuss px vs em afterwards. I do not want to hung up on this at this moment. thanks.